glue/app/qt/application.py

```diff
@@ -36,7 +36,10 @@
 from glue.app.qt.plugin_manager import QtPluginManager
 from glue.app.qt.versions import QVersionsDialog
 from glue.app.qt.terminal import glue_terminal, IPythonTerminalError
-from glue.config import qt_fixed_layout_tab, qt_client, startup_action
+
+from datetime import datetime
```

Comment:
Can you put this import nearer the top of the file? (normally standard library imports come first)